frontend/src/scenes/notebooks/NotebooksList/NotebooksList.tsx

```diff
@@ -4,26 +4,64 @@ import { NotebookListItemType } from '~/types'
 import { Link } from 'lib/lemon-ui/Link'
 import { urls } from 'scenes/urls'
 import { createdAtColumn, createdByColumn } from 'lib/lemon-ui/LemonTable/columnUtils'
-import { LemonButton, LemonInput, LemonTag } from '@posthog/lemon-ui'
-import { notebooksListLogic } from '../Notebook/notebooksListLogic'
-import { useEffect, useMemo, useState } from 'react'
+import {
+    LemonButton,
+    LemonButtonWithDropdown,
+    LemonCheckbox,
+    LemonInput,
+    LemonSelect,
+    LemonTag,
+} from '@posthog/lemon-ui'
+import { DEFAULT_FILTERS, NotebooksListFilters, notebooksListLogic } from '../Notebook/notebooksListLogic'
+import { useEffect } from 'react'
 import { LemonBanner } from 'lib/lemon-ui/LemonBanner'
 import { LemonMenu } from 'lib/lemon-ui/LemonMenu'
 import { IconDelete, IconEllipsis } from 'lib/lemon-ui/icons'
 import { notebookPopoverLogic } from '../Notebook/notebookPopoverLogic'
+import { membersLogic } from 'scenes/organization/Settings/membersLogic'
 
-export function NotebooksTable(): JSX.Element {
-    const { notebooks, notebooksLoading, fuse, notebookTemplates } = useValues(notebooksListLogic)
-    const { loadNotebooks } = useActions(notebooksListLogic)
-    const [searchTerm, setSearchTerm] = useState('')
+function ContainsTypeFilters({
+    filters,
+    setFilters,
+}: {
+    filters: NotebooksListFilters
+    setFilters: (selection: Partial<NotebooksListFilters>) => void
+}): JSX.Element {
+    return (
+        <LemonButtonWithDropdown
+            status="stealth"
+            type="secondary"
+            data-attr={'notebooks-list-contains-filters'}
+            dropdown={{
+                sameWidth: true,
+                closeOnClickInside: false,
+                overlay: [
+                    <>
+                        <LemonCheckbox
+                            size="small"
+                            fullWidth
+                            checked={filters.hasRecordings === true}
+                            onChange={(checked) => {
+                                setFilters({ hasRecordings: checked })
+                            }}
+                            label={'session recording'}
+                        />
+                    </>,
+                ],
+                actionable: true,
+            }}
+        >
+            <span className={'text-muted'}>Notebooks containing...</span>
```

Comment:
I think this should indicate that something is filtered. We could keep it simple and just add an numeric bubble icon to show how many things are selected as well as perhaps a side button to clear the selection like we have in other inputs

Reply:
<img width="331" alt="Screenshot 2023-08-17 at 10 46 38" src="https://github.com/PostHog/posthog/assets/984817/903c5c7e-6311-4370-a8b9-c718486194a6">
<img width="276" alt="Screenshot 2023-08-17 at 10 46 54" src="https://github.com/PostHog/posthog/assets/984817/836a149e-6057-4fd4-94c3-76487ad7270d">
<img width="404" alt="Screenshot 2023-08-17 at 10 47 09" src="https://github.com/PostHog/posthog/assets/984817/60f5e9af-712d-4bbf-b2e0-74788bec389f">

quick change to this...

Actually might be better as plurals now the back end supports either

So it'd be

"Containing any types"
"Containing Insights" or "Containing Feature flags"
"Containing N types"

The names are long so I'd be worried about "Containing Insights, Feature flags, Session Replay" instead of "Containing 3 types"

Alternatively it could be a multi select 🤔

Reply:
![2023-08-17 11 35 36](https://github.com/PostHog/posthog/assets/984817/b00ffb50-0eff-4a79-8b45-0769c63d18da)